cmd/titus-storage/lustre.go

```diff
@@ -70,21 +71,19 @@ func setupLustreMount(parentCtx context.Context, taskID, cname string, vol *core
 	if vm.ReadOnly {
 		flags = flags | MS_RDONLY
 	}
-	mountOptions := append(
-		baseMountOptions,
-		fmt.Sprintf("source=[%s]@tcp:%s", lustreHost, lustreMountName),
-	)
 	cmd.Env = []string{
 		fmt.Sprintf("TITUS_PID1_DIR=%s", pidDir),
 		fmt.Sprintf("MOUNT_TARGET=%s", filepath.Clean(vm.MountPath)),
 		fmt.Sprintf("MOUNT_LUSTRE_HOSTNAME=%s", lustreHost),
+		fmt.Sprintf("MOUNT_LUSTRE_MOUNTNAME=%s", lustreMountName),
 		fmt.Sprintf("MOUNT_FLAGS=%d", flags),
-		fmt.Sprintf("MOUNT_OPTIONS=%s", strings.Join(mountOptions, ",")),
+		fmt.Sprintf("MOUNT_OPTIONS=%s", mountOptions),
 	}
 
+	l.Infof("Running lustre mount command %+v %+v", cmd.Env, cmd)
 	stdoutStderr, err := cmd.CombinedOutput()
 	if err != nil {
-		return fmt.Errorf("Lustre Mount failure: %+v: %s", cmd, string(stdoutStderr))
+		return fmt.Errorf("Lustre Mount failure: %+v: %s (%w)", cmd, string(stdoutStderr), err)
```

Comment:
Any benefit to including the taskID or similar context here?

Reply:
It shouldn't, because this *should* get bubbled up to the titus-storage output, which is logging using the taskid already to the journal, which means this error will (is) visible one click away in the titus UI if you click on "titus-storage"